charts/loki/templates/statefulset.yaml

```diff
@@ -124,5 +124,9 @@ spec:
         requests:
           storage: {{ .Values.persistence.size | quote }}
       storageClassName: {{ .Values.persistence.storageClassName }}
+      {{- if .Values.persistence.selector }}
```

Comment:
Hopefully, we don't forget this local patch during Loki chart upgrades...

Reply:
That's why I opened an upstream PR, I hope it'll be merged.

Reply:
Ha, I see the commit message now :+1: